Tasks/Zhukovets/Task5/Task5.py

```diff
@@ -0,0 +1,38 @@
+li = [1, 2, [2, 4, [[7, 8], [4, 6], 5], [5, 6, [2]]]]
+
+
+def sum_of_list_elem(l: list):
+    sum_ = 0
+    for elem in l:
+        if type(elem) == list:
+            sum_ += sum_of_list_elem(elem)
+        else:
+            sum_ += elem
+    return sum_
+
+
+def fibonacci(n):
+    """func without closure, but it doesn't return the string we need """
+    if n == 1:
+        return 0
+    if n == 2:
+        return 1
+    return fibonacci(n - 1) + fibonacci(n - 2)
```

Comment:
it's possible to return string as well, you just need it as an additional parameter to collect all the values during the reverse recursion